### PR TITLE
[crashtracker] Implement RFC 0005

### PR DESCRIPTION
**Pull request by danielsn**
# What does this PR do?

* Adds a new datatype which implements RFC 0005.
* Adds a translation from the old to the new data-format.
* the `file://` output type now outputs RFC0005 format as well as the old format.

# Motivation

This is step 1 in implementing the RFC.

# Additional Notes

This is step 1 of a multi-step program to move crashtracker over to RFC5.  
For now, both dataformats will co-exist.
Future PRs will switch various parts of the API over to use the new format.
Once this is done, we can retire the old format.

# How to test the change?

Describe here in detail how the change can be validated.

### CODE REVIEW

File: crashtracker/src/rfc5_crash_info/error_data.rs [hunk 1]
```diff
@@ -0,0 +1,64 @@
+use std::collections::HashMap;
```
Comment (gleocadie) [inline crashtracker/src/rfc5_crash_info/error_data.rs:1]: just out of curiosity, shouldn't this be after the header ?

File: crashtracker/src/rfc5_crash_info/stacktrace.rs [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+                    ),
+                    crate::NormalizedAddressMeta::Elf { path, build_id } => (
+                        byte_vec_as_hex(build_id),
+                        Some(BuildIdType::GNU),
+                        Some(FileType::ELF),
+                        Some(path.to_string_lossy().to_string()),
+                    ),
+                    crate::NormalizedAddressMeta::Pdb { path, guid, age } => (
+                        Some(format!(
+                            "{}:{age}",
```
Comment (gleocadie) [inline crashtracker/src/rfc5_crash_info/stacktrace.rs:44]: you can remove the ":".
From what I saw, Windbg will collect the guid and age and concatenate them together without a separator. The request to symbol server is done with the result.

**Review by gleocadie — APPROVED** [review]
Overall looks good to me

File: crashtracker/src/rfc5_crash_info/stacktrace.rs (hunk 2 above)
Comment (danielsn) [reply crashtracker/src/rfc5_crash_info/stacktrace.rs:44]: https://github.com/DataDog/libdatadog/issues/739

File: crashtracker/src/rfc5_crash_info/error_data.rs (hunk 1 above)
Comment (danielsn) [reply crashtracker/src/rfc5_crash_info/error_data.rs:1]: Yes.  OOPS

**Pull request merged**